cypress/support/pages/profile.pageObject.js

```diff
@@ -0,0 +1,39 @@
+import PageObject from '../PageObject';
+
+class ProfilePageObject extends PageObject {
+  get bioLink() {
+    return cy.get('.col-xs-12 > p');
+  }
+
+  get articleOnProfilePage() {
+    return cy.get('h1');
+  }
+
+  get followBtn() {
+    return cy.get('.btn');
```

Comment:
The CSS selector for `followBtn` is very generic ('.btn'). If there are multiple buttons on the page, this could lead to incorrect interactions. Consider using a more specific selector or a data attribute if available.